packages/block-library/src/navigation-link/index.php

```diff
@@ -272,7 +278,7 @@ function register_block_core_navigation_link() {
 	if ( $post_types ) {
 		foreach ( $post_types as $post_type ) {
 			$variation = build_variation_for_navigation_link( $post_type, 'post-type' );
-			if ( 'post' === $variation['name'] || 'page' === $variation['name'] ) {
+			if ( 'post' === $variation['name'] || 'page' === $variation['name'] || 'format' === $variation['name'] ) {
```

Comment:
I wasn't sure what the distinction is between `$built_ins` and `$variations`?

Reply:
This is to force built-ins to show up before variations in the inserter. Feel free to refactor a bit if this is confusing.

Reply:
<img width="329" alt="Screen Shot 2021-03-31 at 10 51 45 AM" src="https://user-images.githubusercontent.com/1270189/113188803-5b5e3f00-920f-11eb-9c86-e7348bd4b62e.png">

Reply:
We did a similar sort in the customizer, but I opted to do a single pass on this, thus the two arrays.

Reply:
Ah, I thought it might be related to the order. I'll add a comment to the code to clarify the purpose 👍